src/glossary.md

```diff
@@ -0,0 +1,281 @@
+# Fuel Glossary
+
+## Address
+
+An address is a cryptographic hash representing an identity of either a wallet, contract address, or a predicate root.
+
+## AssetId
+
+An AssetId is a unique identifier for an on-chain asset. It is defined as the root of the bytecode of the contract minting the asset.
```

Comment:
So the asset ID is basically the contract ID and the contract ID is a _function_ of the root. The actual formula is here: https://fuellabs.github.io/fuel-specs/master/protocol/id/contract.html

Reply:
Oh that's my bad. This sentence was my suggestion.

I guess "It is derived from the root.... " works.